optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg

```diff
@@ -1705,7 +1705,7 @@
            y="150.00014"
            style="font-size:14.66666698px;text-align:center;text-anchor:middle" /></flowRegion><flowPara
          id="flowPara4308-6"
-         style="font-size:14.66666698px;line-height:1.25">Planner works with plain Java objects that are read/written to *.xlsx</flowPara></flowRoot>    <image
+         style="font-size:14.66666698px;line-height:1.25">OptaPlanner works with plain Java objects that are read/written to *.xlsx</flowPara></flowRoot>    <image
```

Comment:
When SVG's are changed, their PNG should be re-exported. Furthermore this increases the size of the text box, so it might affect the layout of the diagram (let's hope it doesn't).
I suggest we meet up for the Inkscape tutorial first and then handle SVG's in a separate PR that also changes the PNG's (not just here, but also in optaplanner-website)?

Reply:
Yes, I didn't consider the text in images. As we move forward, let's consider moving as much text as possible (titles, descriptions) out of the images an into the asciidoc text.

Reply:
Can we remove the SVG change from the PR if it doesn't include the PNG change?
If the SVG file hasn't been opened, there is no guarantee the layout isn't corrupted

Reply:
Adjusted after merge with this change:
https://github.com/kiegroup/optaplanner/commit/277edf1ba7d0d62b96100071c1e1f33c94c77f64